CRM/Logging/ReportDetail.php

```diff
@@ -280,7 +285,25 @@ public function getAllContactChangesForConnection() {
     }
     $this->setDiffer();
     try {
-      return $this->differ->getAllChangesForConnection($this->tables);
+      return $this->differ->getAllChangesForConnection($this->tables, $this->dblimit, $this->dboffset);
+    }
+    catch (CRM_Core_Exception $e) {
+      CRM_Core_Error::statusBounce(ts($e->getMessage()));
```

Comment:
I know this is the same as the existing code lower down but both shouldn't be `ts`'d since there's no actual string text for a translator to look at.

Reply:
Will do. By the way, I've noticed that this approach is being used quite frequently in other functions (different files) as well (all over). I guess one-by-one they should be normalized, right?